public/pages/workflow_detail/agentic_search/agentic_search_workspace.tsx

```diff
@@ -51,6 +62,26 @@ export function AgenticSearchWorkspace(props: AgenticSearchWorkspaceProps) {
   );
   const selectedIndexId = getIn(values, 'search.index.name', '') as string;
 
+  // Get configureAgentOpen from URL params, default to true
+  const queryParams = queryString.parse(location.search);
+  const configureAgentFromUrl = queryParams[CONFIGURE_AGENT_QUERY_PARAM];
```

Comment:
we can make sure here it doesn't return undefined since we are checking for false below on L69
```
String(queryParams[CONFIGURE_AGENT_QUERY_PARAM] ?? '')
```